README.md

```diff
@@ -1,8 +1,8 @@
-[![](https://jitpack.io/v/qiujayen/sticky-layoutmanager.svg)](https://jitpack.io/#qiujayen/sticky-layoutmanager)
+[![](https://jitpack.io/v/ruslansharipov/sticky-layoutmanager.svg)](https://jitpack.io/#ruslansharipov/sticky-layoutmanager)
```

Comment:
need to remove ruslansharipov references since you are merging into qiujayen repo?